components/Transaction/Links.js

```diff
@@ -1,22 +1,25 @@
-import Link from 'next/link'
+import Link from "next/link";
 
 const LinkExplorer = ({ hash }) => (
-    <a href={`https://bithomp.com/explorer/${hash}`} target="_blank" rel="noopener noreferrer">
-        {hash}
-    </a>
-)
+  <a
+    href={`https://bithomp.com/explorer/${hash}`}
```

Comment:
we need to replace old Explorer links to 
new links like 
/tx/
/nft/
/account/
etc